test/test_nn.py

```diff
@@ -3877,6 +3877,60 @@ def test_adaptive_pooling_avg_nhwc_non_contiguous(self):
             self.assertEqual(out, ref_out)
             self.assertEqual(input.grad, ref_input.grad)
 
+    def test_adaptive_pooling_max_nhwc(self):
```

Comment:
Can we convert this test into a test case that with device type? so you can use @onlyCPU, just like what you did here:
https://github.com/pytorch/pytorch/pull/49582/files#diff-7927db349f568afca2de9b94d74ea5c3b8cb468cb6a433d0cc1e61e65c515a36R13793

Reply:
Updated!